src/strings/de.json

```diff
@@ -991,6 +991,7 @@
     "SettingsWarning": "Das Verändern dieser Werte kann Instabilität und Verbindungsprobleme hervorrufen. Wenn Probleme auftreten sollten, empfehlen wir, diese Einstellungen auf die Standardwerte zurückzustellen.",
     "Share": "Teilen",
     "ShowAdvancedSettings": "Erweiterte Einstellungen anzeigen",
+    "ShowDetails": "Details anzeigen",
```

Comment:
Languages other than `en-us` should only be updated with Weblate (after PR merge).
```suggestion
```